server-common/src/test/java/org/apache/kafka/server/common/FeaturesTest.java

```diff
@@ -110,7 +110,7 @@ public void testLatestProductionMapsToLatestMetadataVersion(Features features) {
     @EnumSource(MetadataVersion.class)
     public void testDefaultTestVersion(MetadataVersion metadataVersion) {
         short expectedVersion;
-        if (!metadataVersion.isLessThan(MetadataVersion.IBP_3_8_IV0)) {
+        if (!metadataVersion.isLessThan(MetadataVersion.IBP_3_9_IV0)) {
```

Comment:
Should we change 3.7.-IV4 in the following code to IBP_3_8_0?

`@ClusterTest(types = {Type.KRAFT}, metadataVersion = MetadataVersion.IBP_3_7_IV4)`

`               "SupportedMaxVersion: 4.0-IV0\tFinalizedVersionLevel: 3.7-IV4\t", outputWithoutEpoch(features.get(1)));
`

Reply:
Is the one you're talking about `testDescribeWithKRaftAndBootstrapControllers` in `FeatureCommandTest.java`? I think it would be OK to leave that test case on 3.7-IV4, just to provide some variety.